samples/winapi_drivelist/main.c

```diff
@@ -56,10 +67,32 @@ int main(void)
         debugPrint("%s\n", drive);
         while(*drive++);
     }
-    debugPrint("\ndone");
+    debugPrint("\nDone! Exiting in 10 seconds...");
+
+    Sleep(10000);
+
+    ret = nxUnmountDrive('C');
+    // If there was an error while unmounting (nxUnmountDrive() returns false)
+    if (!ret) {
+      error = GetLastError();
+      errorCount++;
+      debugPrint("Couldn't unmount C: drive Reason: %s\n. Trying to unmount E: drive...\n", error);
+      // Leave some time for the user to read the error
+      Sleep(5000);
+    }
+
+    ret = nxUnmountDrive('E');
+    if (!ret) {
+      error = GetLastError();
+      errorCount++;
```

Comment:
I feel like you should do this before or after printing the message, but not inbetween getting the message and printing it.